### PR TITLE
[WIP] Update pd-overview-planning.md

### LINKED ISSUE #1836: Project Design Chapter
Drafted initial plan for project design chapter (based on Data Study Group process) 

Life cycle of a project 

Initial Considerations & proposal formulation: 
* Capable and suitable partner?
* Research objectives 
* Data 
* Ethics 
* Impact / ensuring findings will be put to meaningful use. 

Getting the right people involved:
* PI?
* Project manager
* Data analyst
* Community expertise 

Proposal to research phases:
* Scoping & defining  
* Data preparation 
* Data access and security 
* Ethics assessment
* Deliverable defining 
* Follow on planning 

Management best practices:
* Agreeing mile stones 
* Regular check ins
* Communication channels 

Who can help?

- Daisy Parry
- Jules Manser 
- Will be opened for review by anyone who is interested

### PR DESCRIPTION
**Pull request by Arielle-Bennett**
### Summary

Fixes #1836 

This is the start of a review of project design guide led by the TPS Programme Management Team but we welcome contributions and feedback from anyone who has ever planned a project! 

### List of changes proposed in this PR (pull-request)

* Add a checklist for project planning 
* Revise text to include more considerations around project planning 
* Add extra links to revised sections or new sections of the book 
* Crossreference with the new chapter on Stakeholder engagement https://github.com/alan-turing-institute/the-turing-way/issues/3105

### What should a reviewer concentrate their feedback on?

- [ ] Anything we've missed from the checklist? 
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by BatoolMM** [comment]
Hi @Arielle-Bennett, I was wondering if I could contribute to this checklist, I don't want to edit/mess up with your PR directly, so I opened a HackMD but am happy to move it to the PR if you prefer that.
https://hackmd.io/7Xdu3wDHR5GdLW9pdIUiVw

**Comment by Arielle-Bennett** [comment]
Same @BatoolMM - can you allow people who are signed in access?

**Comment by BatoolMM** [comment]
Hi @AlexandraAAJ and @Arielle-Bennett - Are you still unable to access the HackMD? I thought I fixed it this morning, but it seems like it didn't go as planned. It wasn't intentional. Could you give it another shot and see if you can access it now? And if you have any comments to add, feel free to do so.

@Arielle-Bennett, Thanks for clarifying that we're good to work on the PR directly. I just want to make sure I'm approaching the sections in the same way you intended. So, if you have any specific guidelines or preferences, please let me know - or if you can clarify in the HackMD that this angel of approaching the sections is fine - happy to move and work directly in the PR.

**Comment by Arielle-Bennett** [comment]
Hey @BatoolMM all fixed now! I've added them into my latest commit alongside some of the stuff I was thinking about - can you take a look tomorrow and let me know what I've missed/if you agree with the additions?

**Comment by BatoolMM** [comment]
> Hey @BatoolMM all fixed now! I've added them into my latest commit alongside some of the stuff I was thinking about - can you take a look tomorrow and let me know what I've missed/if you agree with the additions?

Thank you 🙏 I love _"Useful documentation"_ part in each section. I will go through everything in more detail the 1st thing tomorrow morning!

**Review by AlexandraAAJ — COMMENTED** [review]
This looks very good to me.

File: book/website/project-design/pd-overview/pd-overview-planning.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ Below is a checklist you can use to help identify areas of project planning you
 * Complete any institutional processes for project set up, such as ethics approval or contracting signing. 
 * Useful documentation: RACI matrix, risk register, project charter.
 
+### Stakeholders
+* Identify the project stakeholders
+* Complete a stakeholder mapping, identify level of engagement and stakeholder responsibilities on each stage of the  project
+* Use the facilitating stakeholder engagement for guidance and templates resources[quote the chapter being developed by RAM team #3105]
+
```
[the remark this answers is not in the document]
Comment (BatoolMM) [reply book/website/project-design/pd-overview/pd-overview-planning.md]: I added a few documentation/tools!

File: book/website/project-design/pd-overview/pd-overview-planning.md [hunk 2]
```diff
@@ -1,50 +1,95 @@
...
+* Establish ways of working and collaborative practices for the project team.
+* Identify roles and responsibilities within the project team, using the RACI matrix.
+* Complete any institutional processes for project setup, such as ethics approval or contract signing.
+* Useful documentation: [RACI matrix](https://project-management.com/understanding-responsibility-assignment-matrix-raci-matrix/), [risk register](https://asana.com/resources/risk-register), [project charter](https://www.projectmanager.com/blog/project-charter).
+
+### Stakeholders
+* Identify all individuals, groups, or organizations that have an interest or influence in the project. This includes both internal and external stakeholders
+* Create a visual representation or matrix to understand the relationships between stakeholders and the project. Map their level of engagement, influence, and interest at each stage of the project. This helps prioritise engagement efforts and tailor communication strategies accordingly.
+* Utilise the "Facilitating Stakeholder Engagement" chapter for guidance and template resources [The chapter is being developed by RAM team #3105].
```
Comment (Arielle-Bennett) [inline book/website/project-design/pd-overview/pd-overview-planning.md:51]:
```suggestion
* Utilise the "Facilitating Stakeholder Engagement" chapter for guidance and template resources.
```

We can do a quick update once it's merged instead ;)

File: book/website/project-design/pd-overview/pd-overview-planning.md [hunk 3]
```diff
@@ -1,50 +1,95 @@
...
+* Useful documentation: stakeholder register, stakeholder mapping, stakeholder analysis matrix, [personas](pd-persona).
+
+### Outputs 
+* Plan for the different outputs of your research, such as publications, software, or datasets.
+* Consider licensing and copyright issues for sharing your outputs.
+* Determine how you will manage intellectual property and ownership rights using an IP register.
+* Remember to include any required reporting to funders.
+* Useful documentation: IP register, licensing and copyright statement, data sharing plan.
```
Comment (Arielle-Bennett) [inline book/website/project-design/pd-overview/pd-overview-planning.md:59]: Should just be data management plan again no? Also we should include 
```suggestion
* Useful documentation: [The Turing Way Guide to Communication](cm-comms-overview) IP register, [licensing](rr-licensing) and copyright statement, [data management plan](rr-rdm-dmp).
```

File: book/website/project-design/pd-overview/pd-overview-planning.md [hunk 4]
```diff
@@ -1,50 +1,95 @@
...
+
+### Community & Communications 
+* Identify the target audience for your research.
+* Plan for effective communication and engagement with the community.
+* Consider open and inclusive practices to involve stakeholders in decision-making.
+* Useful documentation: [Code of Conduct](ch-coc), [Contributing Guidelines](cl), [READMEs](https://the-turing-way.netlify.app/collaboration/github-novice/github-novice-firststeps.html?highlight=readmes), communication plan, stakeholder mapping, governance documentation.
```
Comment (Arielle-Bennett) [inline book/website/project-design/pd-overview/pd-overview-planning.md:65]:
```suggestion
* Useful documentation: [Code of Conduct](ch-coc), [Contributing Guidelines](cl), [READMEs](https://the-turing-way.netlify.app/collaboration/github-novice/github-novice-firststeps.html?highlight=readmes), communication plan, stakeholder mapping, [governance documentation](er-ethics-open-source-governance).
```

**Review by Arielle-Bennett — COMMENTED** [review]
Alrighty added a few more links but otherwise I think this can be marked as ready for review!

**Comment by BatoolMM** [comment]
> Alrighty added a few more links but otherwise, I think this can be marked as ready for review!

Fab, I committed all the changes, thank you!!

**Review by BatoolMM — APPROVED** [review]

**Review by AlexandraAAJ — COMMENTED** [review]
Its very easy to read, I like it!

**Pull request merged**